docs/static/spec/openapi/logstash-api.yaml

```
@@ -170,60 +323,210 @@ paths:
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/FlowStats"
+
+  /_node/stats/jvm:
+    get:
+      summary: Get flow statistics
+      description: Return throughput and backpressure details.
+      tags:
+        - node stats
+      responses:
+        "200":
+          description: A JSON object containing jvm statistics.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/JVMStats"
+
+  /_node/stats/os:
+    get:
+      summary: Get Cgroup statistics
+      description: Return a more accurate view of CPU statistics from Cgroup if available.
+      tags:
+        - node stats
+      responses:
+        "200":
+          description: A JSON object containing OS statistics.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  os:
+                    type: object
+                    properties:
+                      cgroup:
+                        type: object
+                        properties:
+                          cpuacct:
+                            type: object
+                            properties:
+                              control_group:
+                                type: string
+                              usage_nanos:
+                                type: integer
+                                format: int64
+                          cpu:
+                            type: object
+                            properties:
+                              control_group:
+                                type: string
+                              cfs_period_micros:
+                                type: integer
+                                format: int64
+                              cfs_quota_micros:
+                                type: integer
+                                format: int64
+                              stat:
+                                type: object
+                                properties:
+                                  number_of_elapsed_periods:
+                                    type: integer
+                                    format: int64
+                                  number_of_times_throttled:
+                                    type: integer
+                                    format: int64
+                                  time_throttled_nanos:
+                                    type: integer
+                                    format: int64
+
+  /_node/stats/pipelines:
+    get:
+      summary: Get pipeline statistics
```

If this endpoint is for multiple pipelines, might be a clearer distinction like this:
```suggestion
      summary: Get statistics for pipelines
```